WTWSMS/events/VIET_Jewish_Flavor_Events.txt

```diff
@@ -62,8 +65,8 @@ character_event = {
 	option = { #I hate them too
 		name = "EVTOPTB1000109021"
 		random_list = {
-			90 = {}
-			10 = {
+			95 = {}
+			5 = {
```

Comment:
Too high. Not liking Jews at your court doesn't suddenly make you hate all Jews or be more likely to expel them. After all, they are still competent employees.